### DISCUSSION TITLE
Do you use Postal in WordPress?

### DISCUSSION THREAD

**Discussion by uhlhosting** [postalserver/postal#2342] [Polls]
Do you use Postal in WordPress?

**Comment by eddywashere** [comment]
I think the postal username format doesn’t pass php mailer validation or whatever default mailer is used. Most of the plugins I tried failed because of that. Would be nice to remove the slash in username.

**Reply by uhlhosting** [reply]
@there is wp_mail function and there is a whole library in postal-php.

**Reply by willpower232** [reply]
You can see in #1498 that Postal never checks the username so I wouldn't worry about what you type in there.

**Reply by uhlhosting** [reply]
I am working on a plugin, if anyone is interested let me know.

**Reply by eddywashere** [reply]
> You can see in #1498 that Postal never checks the username so I wouldn't worry about what you type in there.

Oh wow. I missed that discussion the last time I was checking if that had been requested. I don’t think I tried a username without the slash. Now to find a good ip to spin this up and try again.

**Reply by olegbliaher** [reply]
@uhlhosting I am absolutely interested

**Reply by uhlhosting** [reply]
@olegbliaher > @uhlhosting I am absolutely interested

as soon as I will make it work, will share it open source to all. still hitting some issues.

**Reply by uhlhosting** [reply]
@olegbliaher plugin is in early phase yet it works on sending mails, would you like to test it?

<img width="1679" alt="image" src="https://github.com/postalserver/postal/assets/6324047/fa65bc30-993c-43f0-98ab-1199f1bf8be5">

here is a sneak peak under the hood. its made to work with postal api.

**Comment by uhlhosting** [comment]
First postal api integration plugin for wordpress! Please those interested to test let me know, testing can help solve issues of all kinds before preparing it for wordpress repository.

**Reply by Veeau** [reply]
Hello, I am interested in testing

**Reply by uhlhosting** [reply]
@Veeau 
[postal-mail-main-2.zip](https://github.com/postalserver/postal/files/12270066/postal-mail-main-2.zip)

please feel free to drop improvements, open to all feedback.

**Reply by uhlhosting** [reply]
please note this is still early beta. now working on gitlab-ci for the testing automation.

**Reply by alexk1919** [reply]
> @Veeau [postal-mail-main-2.zip](https://github.com/postalserver/postal/files/12270066/postal-mail-main-2.zip)
> 
> please feel free to drop improvements, open to all feedback.

Hi, is there an updated version of this plugin or another solution to get SMTP working on Wordpress?
I tried FluentSMTP, WP SMTP and 1 more with no success.

Seems Postal isn't accepting the connection.

**Comment by harslannet** [comment]
I finally figured it out! For SMTP clients using CRAM-MD5 authentication (like WP Mail SMTP), the username must be formatted as: {organization-permalink} _ {server-permalink} 
The organization and server permalinks should be joined with an underscore ( _ ). For the password, use the Credential KEY value. This worked for me!
